### PR TITLE
[Plugin] Super Reaction Tweaks

### PR DESCRIPTION
**Pull request by ant0n-0x0000**
Saw a lot of people requesting this today so I tried my best and made this.  As far as I can tell it's fully compliant with the guidelines.

### CODE REVIEW

File: src/plugins/alwaysSuperReact/index.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import definePlugin from "@utils/types";
```
Comment (MrDiamondDog) [inline src/plugins/alwaysSuperReact/index.ts:1]: Add the license at the top, see any other file

File: src/plugins/alwaysSuperReact/index.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import definePlugin from "@utils/types";
+
+export default definePlugin({
+    name: "AlwaysSuperReact",
+    description: "Enables the Super React toggle by default.",
+    authors: [
```
Comment (MrDiamondDog) [inline src/plugins/alwaysSuperReact/index.ts:6]: Add yourself to `Devs`, see other plugins

**Review by MrDiamondDog — CHANGES_REQUESTED** [review]

**Comment by ant0n-0x0000** [comment]
Merged @FieryFlames's super reaction tweaks into the pull request.

File: src/plugins/superReactionTweaks/index.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                match: /(?<=\.useBurstReactionsExperiment.{0,20})useState\(!1\)(?=.+?(\i===\i\.EmojiIntention.REACTION))/,
+                replace: "useState($self.settings.store.superReactByDefault && $1)"
+            }
+        }
+    ],
+    settings,
+
+    shouldPlayBurstReaction(playingCount: number) {
+        if (settings.store.unlimitedSuperReactionPlaying) return true;
+        else if (playingCount <= settings.store.superReactionPlayingLimit) return true;
```
Comment (Vendicated) [inline src/plugins/superReactionTweaks/index.ts:60]:
```suggestion
        if (playingCount <= settings.store.superReactionPlayingLimit) return true;
```

File: src/plugins/superReactionTweaks/index.ts [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        else return false;
```
Comment (Vendicated) [inline src/plugins/superReactionTweaks/index.ts:61]:
```suggestion
        return false;
```

**Review by Vendicated — COMMENTED** [review]
either this or just make it one single return statement with ||

File: src/plugins/superReactionTweaks/index.ts [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+import { definePluginSettings } from "@api/Settings";
+import { Devs } from "@utils/constants";
+import definePlugin, { OptionType } from "@utils/types";
+
+export const settings = definePluginSettings({
+    superReactByDefault: {
+        type: OptionType.BOOLEAN,
+        description: "Reaction picker will default to Super Reactions",
+        default: false,
```
Comment (Vendicated) [inline src/plugins/superReactionTweaks/index.ts:15]:
```suggestion
        default: true,
```

**Comment by Vendicated** [comment]
This plugin needs a README.md file

File: src/plugins/superReactionTweaks/index.ts (hunk 3 above)
Comment (FieryFlames) [reply src/plugins/superReactionTweaks/index.ts:60]: I went with this for verbosity

File: src/plugins/superReactionTweaks/index.ts (hunk 4 above)
Comment (FieryFlames) [reply src/plugins/superReactionTweaks/index.ts:61]: fine

**Comment by ant0n-0x0000** [comment]
Added README.md

File: src/plugins/superReactionTweaks/README.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# Super Reaction Tweaks
+
+This plugin applies configurable various tweaks to super reactions.
+
+![Screenshot](https://user-images.githubusercontent.com/22851444/281598795-58f07116-9f95-4f64-940b-23a5499f2302.png)
+
+### Currently, the plugin allows for two tweaks:
```
Comment (Vendicated) [inline src/plugins/superReactionTweaks/README.md:7]: headers should not skip levels
```suggestion
## Currently, the plugin allows for two tweaks:
```

Comment (Vendicated) [reply src/plugins/superReactionTweaks/README.md]: Also i would just rename this header to 
```suggestion
## Features
```

File: src/plugins/superReactionTweaks/index.ts [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+export default definePlugin({
+    name: "SuperReactionTweaks",
+    description: "Customize the limit of Super Reactions playing at once, and super react by default",
+    authors: [Devs.FieryFlames, Devs.ant0n],
+    patches: [
+        {
+            find: ",BURST_REACTION_EFFECT_PLAY",
+            replacement: {
+                match: /(?<=BURST_REACTION_EFFECT_PLAY:\i=>{.{50,100})(\i\(\i,\i\)>=\d+)/,
```
Comment (Vendicated) [inline src/plugins/superReactionTweaks/index.ts:44]: this doesn't seem quite right. shouldPlayBurstReaction expects a number, not a boolean

image

Comment (FieryFlames) [reply src/plugins/superReactionTweaks/index.ts:44]: to be fair

I wrote that at like 5 am

**Comment by Vendicated** [comment]
cool thanks!

**Pull request merged**